src/style.css

```
@@ -0,0 +1,39 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin: 0 4px 0 0;
+  background: url(images/star.svg) no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin: 0;
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background: url(images/star-active.svg) no-repeat center;
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background: url(images/star-active.svg) no-repeat center;
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background: url(images/star-active.svg) no-repeat center;
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background: url(images/star-active.svg) no-repeat center;
+}
+
+.stars--5 :nth-child(-n + 5) {
+  background: url(images/star-active.svg) no-repeat center;
+}
```

1. we can use as a group
2. no needs to repeat styles that we've already applied
```suggestion
.stars--1 :nth-child(-n + 1),
.stars--2 :nth-child(-n + 2),
.stars--3 :nth-child(-n + 3),
.stars--4 :nth-child(-n + 4),
.stars--5 :nth-child(-n + 5) {
  background: url(images/star-active.svg);
}
```